benchmarks/index.js

```diff
@@ -40,9 +40,9 @@ suite.add(`encode - borc - ${vecLength}`, () => {
 })
 
 suite.add(`encode - stream - borc - ${vecLength}`, () => {
-  const enc = new fastCbor.Encoder({stream (chunk) {
+  const enc = new fastCbor.Encoder({ stream (chunk) {
```

Comment:
please remove all the white space changes, unless they have to be there to make linting pass